mmgen/core/evaluation/eval_hooks.py

```diff
@@ -108,7 +110,8 @@ def after_train_iter(self, runner):
         if rank == 0:
             sys.stdout.write('\n')
             for metric in self.metrics:
-                metric.summary()
+                with torch.no_grad():
```

Comment:
I remember `@torch.no_grad()` has already been added to `summary` function.

Reply:
I just want to make sure the gradient feature map will not be kept. (In case of that users forget to add no_grad() ahead of summary)